src/os/ui/measurebutton.js

```diff
@@ -163,7 +174,7 @@ os.ui.MeasureButtonCtrl.prototype.getMeasureInteraction_ = function() {
 
 
 /**
- * @param {os.ui.action.ActionEvent} evt The event from the menu
+ * @param {os.ui.menu.MenuEvent<Array<number>>} evt The event from the menu
```

Comment:
Just for consistency, please make the handler type match the type provided by the menu. In this case, `os.ui.menu.MenuEvent<undefined>`.